packages/forms-web-app/src/utils/timetables/check-timetable-state.js

```diff
@@ -13,6 +13,8 @@ const hasTimetableDeadlineStarted = (dateTimeDeadlineStart) =>
 	setTimeToStartOfDay(dateTimeDeadlineStart) <= getDateNow() ||
 	isNullSQLDate(new Date(dateTimeDeadlineStart));
 
+const hasDeadlineItemsList = (description) => description?.includes('* ');
```

Comment:
is there an existing function somewhere that parses `description` that we could use? 
i'm assuming `description` gets turned into an array of line items somewhere else in the code (i did a quick search but couldn't find anything so may be wrong).

Reply:
<img width="524" alt="image" src="https://github.com/Planning-Inspectorate/applications-service/assets/15147598/06fa7616-e726-47a7-8e93-015f3b3b31c7">

Its converted to markdown here in `event-view-model.js`, but I felt it probably makes more sense to do this check. within the `check-timetable-state.js` - as thats where we will use it. Or do you mean we should use the same lib `marked` and check if it contains `ul` instead of checking for `*`?

Reply:
ahh right, so it converts the markdown directly into html.. In my head it was going from markdown to json (simple array of strings for the line items), then to html.. my reservation was that trying to check the `description` manually might be error prone (like getting a rogue `*` in there somewhere that isn't a list item). But happy to approve ,we can always revisit if we get some unexpected descriptions coming through that break things

Reply:
Yeah, I had reservations about a rogue `*` as well, for example bolding also uses a `*` but hopefully specifically checking for a `*` followed by a space should mitigate this. With a space, it won't bold so a `* ` should only be for lists. But yeah like you said, if anything happens I can pick it up for a better fix.